WalletWasabi/Legal/Assets/LegalDocumentsWw2.txt

```diff
@@ -78,7 +78,7 @@ I. TERMS AND CONDITIONS
      -The UTXO comes from a One Hop transaction.
 
      
-    BY ACCEPTING THESE TERMS AND CONDITIONS PURSUANT TO SECTION 1.1 YOU EXPRESSLY ACKNOWLEDGE AND AGREE THAT OUR COORDINATOR AUTOMATICALLY DEDUCTS THE TRANSACTION FEES FROM THE TRANSACTION YOU SUBMITTED FOR THE SERVICES IN LINE WITH THE PREVIOUS PARAGRAPHS. YOU EXPRESSLY ACKNOWLEDGE THAT WHILE COINJOINING THE COORDINATOR AUTOMATICALLY SELECTS THE COINS WHICH WILL PARTICIPATE IN THE COINJOIN TRANSACTION.
+    BY ACCEPTING THESE TERMS AND CONDITIONS PURSUANT TO SECTION 1.1 YOU EXPRESSLY ACKNOWLEDGE AND AGREE THAT OUR COORDINATOR AUTOMATICALLY DEDUCTS THE TRANSACTION FEES FROM THE TRANSACTION YOU SUBMITTED FOR THE SERVICES IN LINE WITH THE PREVIOUS PARAGRAPHS. YOU EXPRESSLY ACKNOWLEDGE THAT WHILE COINJOINING THE WASABI CLIENT AUTOMATICALLY SELECTS THE COINS WHICH WILL PARTICIPATE IN THE COINJOIN TRANSACTION.
```

Comment:
> YOU EXPRESSLY ACKNOWLEDGE THAT WHILE COINJOINING THE WASABI CLIENT AUTOMATICALLY SELECTS THE COINS WHICH WILL PARTICIPATE IN THE COINJOIN TRANSACTION.

Should we mention that this is the case only if auto-coinjoin is on?

Reply:
Please revert this.

Reply:
Done. [8b159cb](https://github.com/zkSNACKs/WalletWasabi/pull/8377/commits/8b159cb3a44e7ee9b2d692d1d512fdb3a73a145c)